test/blackbox-tests/test-cases/describe/describe-pp/describe-pp.t/run.t

```diff
@@ -16,4 +16,6 @@ We also make sure that the dump file is not present
 This also works for reason code
 
   $ dune describe pp src/main_re.re
-  ;;Util.log "Hello, world!"
+  # 1 "_build/default/src/main_re.pp.re.ml"
+  # 1 "src/main_re.pp.re"
```

Comment:
How come we're printing the location directives now?

Reply:
the tests have a dedicated `refmt` executable that I changed to match the real `refmt` by printing to stdout